TwitchLib.Client.Models/Builders/ChatCommandBuilder.cs

```diff
@@ -1,67 +0,0 @@
-﻿using System.Collections.Generic;
-
-namespace TwitchLib.Client.Models.Builders
-{
-    public sealed class ChatCommandBuilder : IBuilder<ChatCommand>
```

Comment:
Just wondering, why those two builders were removed?

Reply:
the models for those builders no longer exist
so the better question is why didn't I create a builder for `CommandInfo`? => In my opinion, the model is not very complex and so does not need a builder